R/extract-imagery-metadata_emp.R

```diff
@@ -15,6 +32,23 @@ extract_dataset_id = function (exif) {
 
 #### datatime_local (Format: YYYYMMDD HHMMSS (local time zone, 24 hr)) ####
 
+#' Extract local date and time of image collection
+#'
+#' Pulls the local date and time of image collection (Format: YYYYMMDD HHMMSS) to include in image-level metadata collation
+#'
+#' @param exif$DateTimeOriginal_nocolon local date and time of image collection
+#'
+#' @param datatime_local local date and time of image collection
```

Comment:
The only parameter of this function is `exif` so the definitions of the others should be deleted. The others are just internal variables.